src/main/java/racingcar/domain/car/Car.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.domain.car;
+
+import racingcar.strategy.MoveStrategy;
+
+public class Car {
+    private final Name name;
+    private final Position position;
+
+    public Car(String name) {
+        this.name = new Name(name);
+        this.position = new Position(0);
+    }
+
+    private Car(Name name, int position) {
+        this.name = name;
+        this.position = new Position(position);
+    }
+
+    public Name getName() {
+        return name;
+    }
+
+    public Position getPosition() {
+        return position;
+    }
+
+    public Car move(MoveStrategy moveStrategy) {
+        int nextPosition = position.getPosition();
+        if (moveStrategy.isMove()) {
+            nextPosition++;
+        }
+        return new Car(name, nextPosition);
+    }
```

Comment:
```suggestion
    public Car move(MoveStrategy moveStrategy) {
        if (moveStrategy.isMove()) {
            return new Car(name, position.increase());
        }
        return this;
    }
```

```java
    public Position increase() {
        return new Position(position + INCREASE_SIZE);
    }
```

VO를 만들어내는 이점 중 하나는 내가 원하는 행동을 규정할 수 있다는 것입니다.
_get 하지말고 메세지를 보내라_ 에도 해당할 수 있겠네요 🙂